components/arm/universalrobots/ur.go

```diff
@@ -240,6 +246,9 @@ func (ua *URArm) MoveToPosition(
 ) error {
 	ctx, done := ua.opMgr.New(ctx)
 	defer done()
+	if ua.builtinKinematics {
```

Comment:
[opt] Can world state.geometries/obstacles be nil? Useful to check on that as the logic driving builtnKinematics.

Reply:
I think a logger.Warnf() for now if there are obstacles. So you get a clue as to why it nearly hit your obstacle. And in this case, make the default to false here. 

I think the converse design of checking first and using the built in is more intuitive but it's not that much of a lift to change down the line. Let's do this for now.

Reply:
We should actually go a step further, and in that case, not move at all and return an error to the user.

If an obstacle is explicitly passed, and built-in is specified, then one of the two things that the user passed in is going to get ignored.

We don't actually know which the user cares about more; the obstacle or the built-in kinematics. We shouldn't be picking for them.

Reply:
I am on board with Peter's logic here, I'll change this so that it errors out